core/extensions/impl/crypto_extension.cpp

```diff
@@ -168,4 +187,299 @@ namespace kagome::extensions {
 
     memory_->storeBuffer(out_ptr, common::Buffer(hash));
   }
+
+  // ---------------------- runtime api version 1 methods ----------------------
+
+  runtime::PointerSize CryptoExtension::ext_ed25519_public_keys_v1(
+      runtime::SizeType key_type) {
+    static const common::Buffer error_result{};
+
+    auto &&key_type_id = crypto::decodeKeyTypeId(key_type);
```

Comment:
For what result used as rvalue reference? It makes sense in case function returns cost lvalue reference, but we need to get mutable copy of object (if I remember right, not sure).

Reply:
Just a reflex. "auto &&" results in SomeType& or SomeType&& depending on the right side. This helps to avoid redundant copying. In this case it makes no sense of course.